block/src/Pos/DB/Block.hs

```diff
@@ -114,25 +139,22 @@ putSerializedBlunds
     :: (MonadRealDB ctx m, MonadDB m)
     => NonEmpty (CB.BlockHeader, SerializedBlund) -> m ()
 putSerializedBlunds (toList -> bs) = do
-    let allData = map (\(b,u) -> let hh = sformat hashHexF (headerHash b)
-                                 in ( serialize' b
-                                    , unSerialized u
-                                    , hh
-                                    )
+    let allData = map (\(bh,bu) -> let hh = headerHash bh
```

Comment:
Minor thing: `bu` is a confusing name I think, I think `bl` or `blund` would fit better here.

Reply:
That's how it's called in IOHK's code and I want our code to be similar to their code, even if some things are confusing (as long as they are not too confusing).

Reply:
Fair enough.